packages/terra-form-select/src/terra-dev-site/doc/example/native-select-field/IncompleteField.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import NativeSelectField from 'terra-form-select/lib/native-select/NativeSelectField';
+import styles from '../FormSelectDocCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const IncompleteFieldExample = () => (
+  <NativeSelectField
+    selectId="incomplete-example"
+    label="Incomplete Field Text"
+    required
+    isIncomplete
+    options={[
+      { value: 'volvo', display: 'Volvo' },
+      { value: 'saab', display: 'Saab' },
+      { value: 'mercedes', display: 'Mercedes' },
+      { value: 'audi', display: 'Audi' },
+    ]}
+    placeholder={{}}
```

Comment:
What is this doing? Is placeholder required?

Reply:
I assume the empty object indicates the `-select-` text as the default.

Reply:
The empty object indicates that a placeholder is desired, but no custom values are going to be passed.